.azure-pipelines/azure-pipelines-job-groups.yml

```diff
@@ -50,4 +58,5 @@ jobs:
           displayName: 'JobScript'
       - ${{ if ne(jobGroup.steps, '') }}:
         - ${{ jobGroup.steps }}
+      - ${{ parameters.buildSteps }}
```

Comment:
The buildSteps is run right before the postSteps, you can reuse in the postSteps instead.